config/requirements_core.txt

```diff
@@ -30,7 +28,13 @@ channels==2.2.0                         # Channels; also includes the Daphne ser
 channels_redis==2.4.2                   # Channels Layer
 
 # Misc
-ipython==7.11.*
+ipython==7.15.*
+munkres==1.1.2                          # Algorithm for adjudicator allocation
 redis==3.3.8
 qrcode==6.1                             # QR codes for printed private URL sheets
 html2text==2019.8.11                    # Compatibility with plain-text email clients
+
+# Django admin (Jet)
+# This is an unofficial fork of no-longer-maintained geex-arts/django-jet,
+# patched to support Django 3.
+git+https://github.com/Barukimang/django-jet.git@1.0.8-django3
```

Comment:
@philipbelesky Just drawing attention to this line, which is the patched django-jet

Reply:
Ok. Also to address your comment above, we could always do a fork ourselves if we were concerned about it vanishing or having a breaking update.

Reply:
Yeah, we could. I'd describe my concern as more about the consequence than the probability of it happening—I doubt it'll disappear, but if it does, Tabbycat will become undeployable until we release a patch with our own fork.